packages/core/e2e/fulfillment-process.e2e-spec.ts

```diff
@@ -0,0 +1,238 @@
+/* tslint:disable:no-non-null-assertion */
+import { CustomFulfillmentProcess, FulfillmentState, mergeConfig } from '@vendure/core';
+import { createTestEnvironment } from '@vendure/testing';
+import gql from 'graphql-tag';
+import path from 'path';
+
+import { initialData } from '../../../e2e-common/e2e-initial-data';
+import { TEST_SETUP_TIMEOUT_MS, testConfig } from '../../../e2e-common/test-config';
+
+import { testSuccessfulPaymentMethod } from './fixtures/test-payment-methods';
+import {
+    CreateFulfillment,
+    GetCustomerList,
+    GetOrderFulfillments,
+    TransitFulfillment,
+} from './graphql/generated-e2e-admin-types';
+import { AddItemToOrder } from './graphql/generated-e2e-shop-types';
+import { GET_CUSTOMER_LIST } from './graphql/shared-definitions';
+import { ADD_ITEM_TO_ORDER } from './graphql/shop-definitions';
+import { CREATE_FULFILLMENT, GET_ORDER_FULFILLMENTS, TRANSIT_FULFILLMENT } from './order.e2e-spec';
```

Comment:
If gql operations are needed by more than one spec file, move them to ./graphql/shared-definitions.ts